src/load.py

```diff
@@ -0,0 +1,16 @@
+from xyPlot import AreaStyle 
+from xyPlot import LineStyle 
+from xyPlot import QuantityType 
+from regionToolset import Region 
+from sketch import Sketch 
+from sketch import SketchTransform 
+from xyPlot import SymbolStyle 
+from xyPlot import TextStyle 
+from xyPlot import XYData 
+from xyPlot import XYDataFromFile 
+from xyPlot import XYDataFromFreeBody 
+from xyPlot import XYDataFromHistory 
+from xyPlot import XYDataFromPath 
+from xyPlot import XYDataFromShellThickness 
+from xyPlot import xyDataListFromField 
+# inspected from Abaqus cli
```

Comment:
Please add a new blank line to this file, and also other files

Reply:
Ok, thanks

Reply:
Done on [ad13326](https://github.com/haiiliin/abqpy/pull/1068/commits/ad133265c8ca8b503a1d0f296c26ba7918edb8d9)